lib/compilers/sass.js

```diff
@@ -28,18 +28,13 @@ module.exports = function (raw, cb, compiler, filePath) {
     ? sassOptions.includePaths.concat(paths)
     : paths
 
-  sass.render(
-    sassOptions,
-    // callback for node-sass > 3.0.0
-    function (err, res) {
-      if (err) {
-        cb(err)
-      } else {
-        res.stats.includedFiles.forEach(function (file) {
-          compiler.emit('dependency', file)
-        })
-        cb(null, res.css.toString())
-      }
-    }
-  )
+  try {
+    let res = sass.renderSync(sassOptions)
```

Comment:
`render` is deprecated and [not recommended](https://sass-lang.com/documentation/js-api/functions/render/) if you're using `sass` via npm. I replaced it with `renderSync`, which takes the same `LegacyOptions` param. `renderSync` returns a result or throws an error, so I wrapped everything in a `try..catch` to handle the callbacks.